parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,47 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION == "2.0" ]]; then
+      for dnf_package in cargo opa parted qemu-img moby-runc python3-devel python3-pip kernel-mshv cloud-hypervisor kata-containers moby-containerd-cc mshv-bootloader mshv-linuxloader mshv; do
+        if ! dnf_install 30 1 600 $dnf_package; then
+          exit $ERR_APT_INSTALL_TIMEOUT
+        fi
+      done
+
+      echo "Install UVM build pipeline artifacts"
+      mkdir -p /opt/confidential-containers/share/kata-containers/
+      mv igvm-debug.bin /opt/confidential-containers/share/kata-containers/igvm-debug.bin
+      mv igvm-measurement /opt/confidential-containers/share/kata-containers/igvm-measurement
```

Comment:
You're missing the file `igvm-measurement.cose`, which is the same `igvm-measurement` but signed.

Reply:
I think we also want the following: igvm.bin, igvm-debug-measurement

Reply:
Got it. Updated with binaries added.